src/components/job-common.js

```diff
@@ -172,22 +173,52 @@ export const getTaskCost = ({ vmCostUsd, taskStartTime, taskEndTime }) => {
 };
 
 export const renderTaskCostElement = (cost) => {
-  if (cost === 0.0) {
+  if (cost.toFixed(2) === '0.00') {
     return '< $0.01';
   }
   return `$${cost.toFixed(2)}`;
 };
 
-export const calculateTotalCost = (callObjects) => {
+export const calculateTotalCost = async (callObjects, loadForSubworkflows) => {
   let total = 0;
-  Object.values(callObjects).forEach((call) => {
-    if (!call[0].taskStartTime) {
-      total += 0;
-    } else if (call[0].taskEndTime) {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime, taskEndTime: call[0].taskEndTime });
-    } else {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime });
+  let callObs = callObjects;
+
+  if (!_.isNil || callObjects?.subWorkflowId) {
+    const subWorkflows = await loadForSubworkflows(callObjects.subWorkflowId);
+    callObs = subWorkflows?.calls;
+  }
+
+  for (const call of Object.values(callObs)) {
+    for (const s of call) {
+      const { taskStartTime, taskEndTime, vmCostUsd, subWorkflowId, subWorkflowMetadata } = s;
```

Comment:
can this object be destructured directly in the line above? 

maybe not, I'm not sure, but it might be cleaner if it's possible

Reply:
I don't think so. Because of the way the object is structured, `s` is the object that it can be destructured from.

Reply:
sorry, I'm not being clear. what I mean is something like this:
`for ({ taskStartTime, taskEndTime, vmCostUsd, subWorkflowId, subWorkflowMetadata } of call) { ... }`

which seems to work in my quick node test

Reply:
I see now. Is there a particular benefit to destructuring in the line above? I structured it this way because it was a little easier to read as I was coding up this particular function. But maybe it's not?

Reply:
I disagree, but I guess it's a matter of opinion. I'm not going to fight you on it